templates/data_import/index.html

```diff
@@ -19,7 +19,7 @@
 
     <h2>
       Uploads
-      <a role="button" class="btn btn-outline-primary float-right" href="/data-import/upload-standardized-file/">
+      <a role="button" class="btn btn-outline-primary float-right" href="/admin/data_import/standardizedfile/add/">
```

Comment:
FWIW, we're ultimately going to deprecate this view, so this change isn't strictly necessary, but you don't need to revert it.